astropy/modeling/core.py

```diff
@@ -1289,6 +1563,10 @@ def _format_str(self, keywords=[]):
 
         param_table = Table(columns, names=self.param_names)
```

Comment:
Maybe use `QTable` here, so that the parameters are actually stored as quantities? But that would mean changing the logic above a bit, since one cannot set a unit of a quantity directly (as is done below).
Perhaps easiest to leave it to a comment for now.